src/Formatter/ArrayBasedDeformatter.php

```diff
@@ -10,6 +10,7 @@
 use Crell\Serde\SerdeError;
 use Crell\Serde\TypeCategory;
 use Crell\Serde\TypeMismatch;
+use function array_key_exists;
```

Comment:
For stdlib functions, do not `use` them.  Use a `\` prefix on them when used in code.

Reply:
Fixed